lib/beaker/host/unix/exec.rb

```diff
@@ -26,7 +26,7 @@ def get_ip
     if self['platform'].include?('solaris') || self['platform'].include?('osx')
       execute("ifconfig -a inet| awk '/broadcast/ {print $2}' | cut -d/ -f1 | head -1").strip
     else
-      execute("ip a|awk '/global/{print$2}' | cut -d/ -f1 | head -1").strip
+      execute("ip a | awk '/global/{print$2}' | cut -d/ -f1 | #{self['hypervisor'] == 'vagrant' ? 'tail' : 'head'} -1").strip
```

Comment:
I'm just kinda doing a drive-by review here, so I don't have any context, but....

Are we sure we want to *not* do this for solaris and osx platforms?

Maybe the answer is that solaris and osx aren't supported in the vagrant hypervisor so yes, we don't need to do this for them.